### PR TITLE
Fixed blurry icons for zoom and warning in the gizmo overlay

### PR DESCRIPTION
**Pull request by julienamsellem**
### Purpose of this PR
https://fogbugz.unity3d.com/f/cases/1296206/

Probably because of custom screen settings (DPI > 100%) some icons were blurry (see screenshot below)
VFX (1)

The fix is to properly pick the @2x icons when the DPI > 100% (see the screenshot below)
Unity_cGq4FvEwz7

I did the same fix for the warning icon even if the difference was less visible
Unity_L26ZK7uLcj2

---
### Testing status    
Tested on 150% DPI screen => no blurry effect thanks to 32x32px icon    
Tested on 100% DPI screen => no blurry effect thanks to trilinear filtering
Tested on dark and light themes
 
---
### Comments to reviewers
We should probably refine the lower resolution icon to try to reduce the blurry effect
=> Thanks to trilinear filtering even the lower resolution icon looks good on screen

### CODE REVIEW

File: com.unity.visualeffectgraph/Editor/Inspector/VFXSlotContainerEditor.cs [hunk 1]
```diff
@@ -279,8 +279,8 @@ public class Contents
         public static GUIContent name = EditorGUIUtility.TrTextContent("Name");
         public static GUIContent type = EditorGUIUtility.TrTextContent("Type");
         public static GUIContent mode = EditorGUIUtility.TrTextContent("Mode");
-        public static GUIContent gizmoLocalWarning = EditorGUIUtility.TrIconContent(EditorGUIUtility.Load(EditorResources.iconsPath + "console.warnicon.sml.png") as Texture2D, "Local values require a target GameObject to display");
-        public static GUIContent gizmoIndeterminateWarning = EditorGUIUtility.TrIconContent(EditorGUIUtility.Load(EditorResources.iconsPath + "console.warnicon.sml.png") as Texture2D, "The gizmo value is indeterminate.");
+        public static GUIContent gizmoLocalWarning = EditorGUIUtility.TrIconContent(EditorGUIUtility.LoadIconForSkin(EditorResources.iconsPath + "console.warnicon.sml.png", EditorGUIUtility.skinIndex), "Local values require a target GameObject to display");
```
Comment (ludovic-theobald) [inline com.unity.visualeffectgraph/Editor/Inspector/VFXSlotContainerEditor.cs:282]: I see there exists a method LoadIcon(string name) that does the same thing, but without requiring a second argument. Did you miss it or is there another reason for setting the skin explicitly ?

File: com.unity.visualeffectgraph/Editor/Inspector/VFXSlotContainerEditor.cs [hunk 2]
```diff
@@ -301,8 +301,10 @@ static Styles()
             warningStyle.margin.right = 1;
 
             frameButtonStyle = new GUIStyle();
-            frameButtonStyle.normal.background = EditorGUIUtility.Load(EditorResources.iconsPath + "d_ViewToolZoom.png") as Texture2D;
-            frameButtonStyle.active.background = EditorGUIUtility.Load(EditorResources.iconsPath + "d_ViewToolZoom On.png") as Texture2D;
+            frameButtonStyle.normal.background = EditorGUIUtility.LoadIconForSkin(EditorResources.iconsPath + "ViewToolZoom.png", EditorGUIUtility.skinIndex);
+            frameButtonStyle.active.background = EditorGUIUtility.LoadIconForSkin(EditorResources.iconsPath + "ViewToolZoom On.png", EditorGUIUtility.skinIndex);
+            frameButtonStyle.normal.background.filterMode = FilterMode.Trilinear;
+            frameButtonStyle.active.background.filterMode = FilterMode.Trilinear;
```
Comment (ludovic-theobald) [inline com.unity.visualeffectgraph/Editor/Inspector/VFXSlotContainerEditor.cs:307]: I don't know if the icon texture have mipmaps, if they don't, then FilterMode.Bilinear is enough. Anyways, better safe then sorry, so Trilinear is fine to me :)

File: com.unity.visualeffectgraph/Editor/Inspector/VFXSlotContainerEditor.cs (hunk 1 above)
Comment (julienamsellem) [reply com.unity.visualeffectgraph/Editor/Inspector/VFXSlotContainerEditor.cs]: I missed it, good catch ! Thank you :)

**Review by ludovic-theobald — APPROVED** [review]

File: com.unity.visualeffectgraph/Editor/Inspector/VFXSlotContainerEditor.cs (hunk 2 above)
Comment (julienamsellem) [reply com.unity.visualeffectgraph/Editor/Inspector/VFXSlotContainerEditor.cs:307]: Yep, for two icons I'm not worried about the cost of trilinear filtering and feel safe to the visual quality.

**Review by VitaSkruibyte — APPROVED** [review]
Julien's tests are perfect 💯, skipping QA pass. Thank you!

**Pull request merged**